Cargo.toml

```
@@ -4,25 +4,46 @@ version = "0.1.0"
 authors = ["Scott Mabin <scott@mabez.dev>", "Arjan Mels <arjan@mels.email>"]
 edition = "2018"
 
+
+# Allow overriding of memcpy, memset, etc.
+[package.metadata.cargo-xbuild]
+memcpy = false
+
+
+[features]
+default=["external_ram", "mem"]
```

```suggestion
default=["mem"]
```
I don't think we should enable psram by default, as most esp32's do not come with it.